go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/temporalio/s2s-proxy
 
-go 1.23.2
-
-toolchain go1.23.4
```

Comment:
The `toolchain` version is only present here if it differs from the `go` version (on L3).

I don't see a reason they need to be different.